openpype/hosts/nuke/api/lib.py

```diff
@@ -1795,3 +1799,65 @@ def recreate_instance(origin_node, avalon_data=None):
             dn.setInput(0, new_node)
 
     return new_node
+
+
+class NukeDirmap(HostDirmap):
+    def __init__(self, host_name, project_settings, sync_module, file_name):
+        """
+            Args:
+                host_name (str): Nuke
+                project_settings (dict): settings of current project
+                sync_module (SyncServerModule): to limit reinitialization
+                file_name (str): full path of referenced file from workfiles
+        """
+        self.host_name = host_name
+        self.project_settings = project_settings
+        self.file_name = file_name
+        self.sync_module = sync_module
+
+        self._mapping = None  # cache mapping
+
+    def on_enable_dirmap(self):
+        pass
+
+    def dirmap_routine(self, source_path, destination_path):
```

Comment:
could we make sure that what ever slashes in path will come from settings will be normalized to forward slashes - nuke is internally using only those even it is on Windows